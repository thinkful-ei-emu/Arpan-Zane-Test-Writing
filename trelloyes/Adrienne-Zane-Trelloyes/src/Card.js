import React from 'react';

export default function Card(props) {
  console.log(props.title);
  return (
    <div className="Card">
      <button type="button">Delete</button>
      <h3>{props.title}</h3>
      <p>{props.content}</p>
    </div>
  )
}