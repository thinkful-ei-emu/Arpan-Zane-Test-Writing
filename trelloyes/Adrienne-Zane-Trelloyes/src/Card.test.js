import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import renderer from 'react-test-renderer';




describe('<Card/>',()=>{
    it('renders without crashing',()=>{
        const div= document.createElement('div');

        ReactDOM.render(<Card title="Card 1" content="Stuffs and Things"/>,div);


        ReactDOM.unmountComponentAtNode(div);
    });
    it('renders this UI as Expected',()=>{
        const tree=renderer.create(<Card title="Card 1" content="Stuffs And Things"/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});


