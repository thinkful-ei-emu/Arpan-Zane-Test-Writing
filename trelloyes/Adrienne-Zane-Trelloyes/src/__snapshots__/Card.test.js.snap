// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card/> renders this UI as Expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    Delete
  </button>
  <h3>
    Card 1
  </h3>
  <p>
    Stuffs And Things
  </p>
</div>
`;
